services/core-web/src/components/Forms/noticeOfWork/VerifyNoWContacts.js

```diff
@@ -48,211 +87,264 @@ const renderContacts = ({
-                        onClick={() => {
-                          fields.remove(index);
-                        }}
-                        className="position-right no-margin"
-                      >
-                        <img name="remove" src={TRASHCAN} alt="Remove MineType" />
-                      </Button>
-                    )}
-                  </div>
-                }
-                bordered={false}
-              >
-                <Row align="middle" justify="center">
-                  <Col span={9}>
-                    <h4>
-                      {contactExists ? startCase(fields.get(index).party.name) : "New Contact"}
-                    </h4>
-                    {contactExists && (
-                      <div>
-                        <div className="inline-flex">
-                          <div className="padding-right">
-                            <MailOutlined className="icon-sm" />
-                          </div>
-                          {fields.get(index).party.email &&
-                          fields.get(index).party.email !== "Unknown" ? (
-                            <a href={`mailto:${fields.get(index).party.email}`}>
-                              {fields.get(index).party.email}
-                            </a>
-                          ) : (
-                            <p>{Strings.EMPTY_FIELD}</p>
-                          )}
+      <Col span={8}>
+        <Row className="contact-rows">
+          <Col span={24} style={{ minHeight: "130px" }}>
+            <h3>Application Contacts</h3>
+            <p>
+              Contacts listed here come from the original Notice of Work. Click &quot;Search
+              Contact&quot; to see a list of possible CORE matches in the &quot;Matching contact
+              options&quot; column.
+            </p>
+          </Col>
+          {fields.map((field, index) => {
+            // eslint-disable-next-line no-unused-expressions
+            fields.get(index).id
+              ? (fields.get(index).id = fields.get(index).id)
+              : (fields.get(index).id = uuidv4());
+            const contactExists = fields.get(index) && !isEmpty(fields.get(index).party);
+
+            const isSelectedContact = selectedContactIndex === index;
+            const selectedCorePartyGuid = contactFormValues
+              .filter(({ id }) => id === fields.get(index).id)
+              .map(({ party_guid }) => party_guid)[0];
+            const selectedCoreParty = selectedData
+              .filter(({ value }) => selectedCorePartyGuid === value)
+              .map((contact) => contact)[0];
+            const contactInformation = selectedCoreParty ? selectedCoreParty : fields.get(index);
+            return (
+              <Col span={24} key={fields.get(index).id}>
+                <Card
+                  hoverable
+                  style={
+                    isSelectedContact ? { boxShadow: "rgb(124 102 173) 0px 0px 10px 3px" } : {}
```

Comment:
Should we add this to the `ant-card-now` class?
```
&-selected {
...
}
```

Reply:
updated!